.github/dependabot.yml

```diff
@@ -2,4 +2,5 @@ version: 2
 updates:
   - package-ecosystem: "npm"
     directory: /
-    schedule: daily
+    schedule:
+      interval: daily
```

Comment:
I guess dependabot CI (new thing I think?) only runs on master.
They don't have their YAML schema stuff for this file yet (like they do for workflows!)